database/models/user.js

```diff
@@ -3,92 +3,95 @@
 /* eslint-disable func-names */
 
 module.exports = (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    email: {
-      type: DataTypes.STRING,
-      required: true,
-      unique: true,
-    },
-    password: {
-      allowNull: false,
-      type: DataTypes.STRING,
-    },
-    firstName: {
-      allowNull: false,
-      type: DataTypes.STRING
-    },
-    lastName: {
-      allowNull: false,
-      type: DataTypes.STRING
-    },
-    verificationToken: DataTypes.STRING,
+  const User = sequelize.define(
+    'User',
+    {
+      email: {
+        type: DataTypes.STRING,
+        required: true,
+        unique: true
+      },
+      password: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      firstName: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      lastName: {
+        allowNull: false,
+        type: DataTypes.STRING
+      },
+      verificationToken: DataTypes.STRING,
 
-    isVerified: {
-      defaultValue: false,
-      type: DataTypes.BOOLEAN
-    },
-    type: {
-      type: DataTypes.ENUM('admin', 'user'),
-      defaultValue: 'user',
+      isVerified: {
+        defaultValue: false,
+        type: DataTypes.BOOLEAN
+      },
+      type: {
+        type: DataTypes.ENUM('admin', 'user'),
+        defaultValue: 'user'
+      }
     },
-
-  }, {});
+    {}
+  );
   User.associate = function(models) {
     User.hasOne(models.Profile, {
       foreignKey: 'userId',
       as: 'profile',
       onDelete: 'CASCADE',
-      onUpdate: 'CASCADE',
+      onUpdate: 'CASCADE'
```

Comment:
Please put back the trailing commas you removed from this file as they are important.

Note: Don't go making changes to files that have nothing to do with what you are working on.

Reply:
Done